clients/java/src/main/java/io/camunda/zeebe/client/impl/command/UpdateUserTaskCommandImpl.java

```diff
@@ -74,70 +74,66 @@ public UpdateUserTaskCommandStep1 action(final String action) {
   @Override
   public UpdateUserTaskCommandStep1 dueDate(final String dueDate) {
     ArgumentUtil.ensureNotNull("dueDate", dueDate);
-    getChangesetEnsureInitialized().put(Changeset.JSON_PROPERTY_DUE_DATE, dueDate);
```

Comment:
🔧 Those changes need to stay, I think. We adjusted the model, so this won't work otherwise with the `Zeebe...` classes anymore.

Reply:
As synced: let's rollback those changes as it is here now and also use V1 REST YAML to generate the "old-rest" data model.

Reply:
As synced (PT.2): We don't need to rollback those changes

Reply:
Back and forth, love it 😆